### PR TITLE
W-13828786-IAM-se

### PR DESCRIPTION
**Pull request by sebermuley**

### CODE REVIEW

File: rpa-manager/modules/ROOT/pages/processevaluation-approve.adoc [hunk 1]
```diff
@@ -42,10 +43,9 @@ After approval, the process is also displayed either in the personal backlog of
...
 == See Also
 
-* xref:processevaluation-overview.adoc[Process Evaluation]
 * xref:processevaluation-consider.adoc[Considering Which Processes to Automate]
-* xref:processevaluation-configure.adoc[Configuring Process Evaluations]
-* xref:processevaluation-propose.adoc[Proposing a Process Candidate for Automation]
-//* xref:processevaluation-approve.adoc[Approving a Process for Automation]
-
+* xref:rpa-home::anypoint-migration-overview.adoc[Anypoint Platform Migration Overview]
+* https://docs.mulesoft.com/access-management/[Anypoint Access Management]
```
Comment (sarathecoubian) [inline rpa-manager/modules/ROOT/pages/processevaluation-approve.adoc]:
```suggestion
* https://docs.mulesoft.com/access-management/[Access Management]
```

File: rpa-manager/modules/ROOT/pages/index.adoc [hunk 2]
```diff
@@ -16,13 +17,23 @@ Ensure that access to this URL is possible from your organization.
...
+MuleSoft RPA is integrating with Anypoint Platform to unify the login experience and provide support for external identity providers (IdPs). 
+
+Refer to xref:rpa-home::anypoint-migration-overview.adoc[Anypoint Platform Migration Overview] for an overview of the changes and requirements.
+
+If you are unsure, whether your org is migrated, open the *Home* module in RPA Manager:
+
+* Pre-migration, there is a quick link to Composer:
++ 
+image:rpa-manager-home-premigration.png[Screenshot of the Composer quick link panel in the Home module of RPA Manager,25%,25%]
+* Post-migration, there is a quick link to Anypoint Access Management:
```
Comment (sarathecoubian) [inline rpa-manager/modules/ROOT/pages/index.adoc]:
```suggestion
* Post-migration, there is a quick link to Access Management:
```

File: rpa-manager/modules/ROOT/pages/index.adoc [hunk 3]
```diff
@@ -16,13 +17,23 @@ Ensure that access to this URL is possible from your organization.
...
++
+image:rpa-manager-home-postmigration.png[Screenshot of the Composer quick link panel in the Home module of RPA Manager,25%,25%]
 
 == See Also
 
+* xref:rpa-home::anypoint-migration-overview.adoc[Anypoint Platform Migration Overview]
+* xref:https://docs.mulesoft.com/access-management/[Anypoint Access Management]
```
Comment (sarathecoubian) [inline rpa-manager/modules/ROOT/pages/index.adoc]:
```suggestion
* xref:https://docs.mulesoft.com/access-management/[Access Management]
```

File: rpa-manager/modules/ROOT/pages/processevaluation-approve.adoc [hunk 4]
```diff
@@ -42,10 +43,9 @@ After approval, the process is also displayed either in the personal backlog of
...
+* https://docs.mulesoft.com/access-management/teams[Anypoint Access Management - Manage User Access Using Teams]
```
Comment (sarathecoubian) [inline rpa-manager/modules/ROOT/pages/processevaluation-approve.adoc]:
```suggestion
* https://docs.mulesoft.com/access-management/teams[Access Management - Manage User Access Using Teams]
```

File: rpa-manager/modules/ROOT/pages/settings-adminsettings-general.adoc [hunk 5]
```diff
@@ -0,0 +1,39 @@
+= Configure General Settings
+:page-notice-banner-message: MuleSoft RPA is integrating with Anypoint Platform to unify the login experience and provide support for external identity providers (IdPs). After your organization migrates to Anypoint Platform, the Settings module replaces the Organization Management module.
+
+The *General* tab in the *Admin Settings* view enables you to configure the currency and decimal sign to show in RPA Manager modules.
+
+== Before You Begin
+
+* Ask an Anypoint Access Management administrator to assign you the required permissions:
```
Comment (sarathecoubian) [inline rpa-manager/modules/ROOT/pages/settings-adminsettings-general.adoc]:
```suggestion
* Ask an organization administrator in Access Management to assign you the required permissions:
```

File: rpa-manager/modules/ROOT/pages/settings-adminsettings-reassignassets.adoc [hunk 6]
```diff
@@ -0,0 +1,80 @@
...
+The following assets are deleted on the deletion of the user:
+
+* Private dashboards
+* User API Keys
+
+User Tasks are not shown in the *Reassign Assets* view and can be reassigned via the *Unprocessed User Tasks* view.
+
+== Before You Begin
+
+* Ask an Anypoint Access Management administrator to assign you the required permissions:
```
Comment (sarathecoubian) [inline rpa-manager/modules/ROOT/pages/settings-adminsettings-reassignassets.adoc]:
```suggestion
* Ask an organization administrator in Access Management to assign you the required permissions:
```

File: rpa-manager/modules/ROOT/pages/settings-connect.adoc [hunk 7]
```diff
@@ -0,0 +1,83 @@
+= Connecting to RPA Processes from Salesforce Products
+:page-notice-banner-message: You can now use Connected Apps for a more secure and scalable authentication method than User API Keys.
+
+RPA Manager requires authentication to integrate RPA processes into other Salesforce products. For example, you can use your API key to configure an RPA connector in a MuleSoft Composer flow.
+
+For security reasons, you can only see and copy an API key on creation.
+
+== Before You Begin
+
+* Ask an Anypoint Access Management administrator to assign you the required permissions:
```
Comment (sarathecoubian) [inline rpa-manager/modules/ROOT/pages/settings-connect.adoc]:
```suggestion
* Ask an organization administrator in Access Management to assign you the required permissions:
```

File: rpa-manager/modules/ROOT/pages/usermanagement-manage.adoc [hunk 8]
```diff
@@ -1,4 +1,5 @@
 = Managing Users
+:page-notice-banner-message: MuleSoft RPA is integrating with Anypoint Platform to unify the login experience and provide support for external identity providers (IdP). After your organization is migrated to Anypoint Platform, the User Management module is removed and the RPA Administrator manages the users in Anypoint Access Management.
```
Comment (sarathecoubian) [inline rpa-manager/modules/ROOT/pages/usermanagement-manage.adoc]:
```suggestion
:page-notice-banner-message: MuleSoft RPA is integrating with Anypoint Platform to unify the login experience and provide support for external identity providers (IdP). After your organization is migrated to Anypoint Platform, the User Management module is removed and the RPA Administrator manages the users in Access Management.
```

**Review by sarathecoubian — DISMISSED** [review]

File: rpa-manager/modules/ROOT/pages/index.adoc [hunk 9]
```diff
@@ -16,13 +17,23 @@ Ensure that access to this URL is possible from your organization.
...
 
-//== Get started
+== IAM Migration
 
-// A table with roles and starting modules for these roles?
+MuleSoft RPA is integrating with Anypoint Platform to unify the login experience and provide support for external identity providers (IdPs). 
+
+Refer to xref:rpa-home::anypoint-migration-overview.adoc[Anypoint Platform Migration Overview] for an overview of the changes and requirements.
+
+If you are unsure, whether your org is migrated, open the *Home* module in RPA Manager:
```
Comment (bjoson-mulesoft) [inline rpa-manager/modules/ROOT/pages/index.adoc]:
```suggestion
If you are unsure whether your org is migrated, open the *Home* module in RPA Manager:
```

File: rpa-manager/modules/ROOT/pages/settings-connect.adoc [hunk 10]
```diff
@@ -0,0 +1,83 @@
...
+
+|Create, edit, and delete API keys
+|RPA Administrator
+|User API Key Create, User API Key Edit, User API Key Delete
+
+|===
+
+* Users must have an Anypoint account to associate the API key with.
+* You must have a secure way to transfer the API Key to the user.
+* Users must have a secure place accessible to store their API key.
```
Comment (bjoson-mulesoft) [inline rpa-manager/modules/ROOT/pages/settings-connect.adoc]:
```suggestion
* Users must have a secure place to store their API key.
```

File: rpa-manager/modules/ROOT/pages/settings-connect.adoc [hunk 11]
```diff
@@ -0,0 +1,83 @@
...
+. Click *Create*.
+. [[form-createuserapikey]] Complete the form *Create User API Key*:
+* *User*:
++
+Select a user whose identity and permissions are used to connect from another Salesforce products to RPA Manager.
+* *Expiration Date*:
++
+Select an expiration date for the user API key from the calendar.
+. Click *OK*.
+. In the window *Key Created!*, click *Copy to clipboard*.
```
Comment (bjoson-mulesoft) [inline rpa-manager/modules/ROOT/pages/settings-connect.adoc]:
```suggestion
. In the window *Key Created*, click *Copy to clipboard*.
```

Comment (bjoson-mulesoft) [reply rpa-manager/modules/ROOT/pages/settings-connect.adoc]: Our standard is to exclude punctuation in names or label references.
See https://confluence.internal.salesforce.com/display/MTDT/MuleSoft+CX+Writing+Style+Reference, under UI elements.

Comment (sebermuley) [reply rpa-manager/modules/ROOT/pages/settings-connect.adoc]: Created bug [W-13871084](https://gus.lightning.force.com/a07EE00001XRYfSYAX) for dev

Comment (sebermuley) [reply rpa-manager/modules/ROOT/pages/settings-connect.adoc]: and removed ! in docs

**Comment by sebermuley** [comment]
Closed https://github.com/mulesoft/docs-rpa/pull/214 (included changes here)

**Review by Cristian-Venticinque — APPROVED** [review]

**Pull request merged**